packages/react-sdk/src/state/export/exportWhiteboard.test.ts

```diff
@@ -93,11 +100,23 @@ describe('convertWhiteboardToExportFormat', () => {
       addElement0(doc);
     });
 
-    jest.mocked(convertBlobToBase64).mockImplementation(async (blob: Blob) => {
-      const text = await readBlobAsText(blob);
-      // ensure that the response data is passed into the function
-      expect(text).toEqual('image content');
-      return btoa('encoded image content');
+    vi.spyOn(lib, 'convertBlobToBase64').mockImplementation(
```

Comment:
Mocked at top of file

```suggestion
    vi.mocked(lib, 'convertBlobToBase64').mockImplementation(
```